rules/helm/def.bzl

```diff
@@ -63,6 +71,67 @@ def package(**kwargs):
         **kwargs
     )
 
+def _dependencies_impl(ctx):
```

Comment:
This is a bit hard to follow, is it generating a BUILD.bazel in place? I cannot see where actually it is consumed, could you please explain me how this works?

Reply:
This rule is not consumed in their very same PR. I will be first used in the Eirini PR that is introducing some sub-charts.

Reply:
The workspace created with the repository_rule can then be used in the kubecf package with `subcharts = ["@kubecf_helm_dependencies//charts"],`.